packages/goto/src/index.js

```diff
@@ -158,6 +159,19 @@ module.exports = ({
 
     const prePromises = []
 
+    if (abortTypes.length > 0) {
+      await page.setRequestInterception(true)
+      page.on('request', req => {
+        const resourceType = req.resourceType()
+        if (!abortTypes.includes(resourceType)) {
+          debug('continue', { url: req.url(), resourceType })
+          return req.continue(req.continueRequestOverrides(), 2)
```

Comment:
Sorry if I am meddling where I don't belong :). This `.continue({}, 2)` will cause the request to continue even if `adBlocker` calls `.abort('..',1)`. I think your intent might be to defer to other handlers by using `.continue({}, 0)` since `abortTypes` has no opinion about it?

Reply:
yes, that was the intention; should I call `isInterceptResolutionHandled` to be sure about that?

Reply:
When #7796 is approved (I hope soon), that method will be available and should definitely be used. 

Also see the PR for a discussion on when to use a custom priority to continue a request. 

Again sorry to jump in, I just found this issue and thought it was interesting because I’m learning a lot by how people are implementing support for cooperative mode.

Reply:
no problem 🙂 I did my best implementing the API and since there is not too examples out there probably the code can be rewritten in a better way

Reply:
If you decide to change anything, I suggest using `.continue({...},0)` for this case :)